resources/language/resource.language.pt_pt/strings.po

```diff
@@ -185,7 +185,7 @@ msgstr "Por favor, aguarde..."
 
 msgctxt "#30111"
 msgid "Getting results from providers..."
-msgstr "A obter resultados dos provedores..."
+msgstr "Obtendo resultados dos provedores..."
```

Comment:
Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)